Tangem/Modules/LegacyTokenList/LegacyTokenListViewModel.swift

```diff
@@ -338,11 +355,21 @@ private extension LegacyTokenListViewModel {
     }
 
     func isTokenAvailable(_ tokenItem: TokenItem) -> Bool {
+        guard let settings = mode.settings else {
+            return false
+        }
+
         if case .token(_, let blockchain) = tokenItem,
            case .solana = blockchain,
-           mode.settings?.longHashesSupported == false {
+           settings.longHashesSupported == false {
             return false
         }
+
+        if case .chia = tokenItem.blockchain,
+           !settings.existingCurves.contains(tokenItem.blockchain.curve) {
```

Comment:
а нужно только по chia ?
может сделать 
```suggestion
        if !settings.existingCurves.contains(tokenItem.blockchain.curve) {
```
и тип если вообще такой кривой на карте нет, то и усе

Reply:
Страшно было для всех , потому что тогда как себя вести будут другие валюты ...

Reply:
@tureck1y Тут тебе надо отсоветовать делать для всех или нет, но кажется такие штуки в релизную ветку гнать не очень

Reply:
ну текстовые если говорит только про чиа, то не надо для всех

Reply:
> ну текстовые если говорит только про чиа, то не надо для всех

Убрал проверку для Chia